.travis.yml

```diff
@@ -31,7 +32,13 @@ jobs:
       env: TEST=clang-tidy-fix
            BEFORE_DOCKER_SCRIPT="source moveit_kinematics/test/test_ikfast_plugins.sh"
            CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unused-parameter -Wno-unused-function -Wno-overloaded-virtual"
-#           ROS_REPO=ros-shadow-fixed
+    - if: branch =~ /^(.*-devel|master)$/
```

Comment:
Should this really be part of this PR?

Reply:
These changes are also in my bug fixes PR and I cherry-picked them here to unify travis runs on melodic-devel.  Whichever one gets merged first I'll drop them out of the other PR.  

Would you rather wait on the other change to be merged and then I'll drop them out of here as they are unrelated to this change?